### PR TITLE
landing page: Fix subjects display error

### LINKED ISSUE #1039: landing page: no keywords, yet header is displayed
Screenshot 2021-08-03 at 13 56 46

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes #1039

image

### CODE REVIEW

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     </div>
   </div>
   {%- endfor %}
-  {% set keywords = subjects|rejectattr("scheme") %}
+  {% set keywords = subjects|rejectattr("scheme")|list %}
```
Comment (fenekku) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html:25]: Huh... what was it returning before?

**Review by fenekku — APPROVED** [review]
Nice, I am just curious.

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html (hunk 1 above)
Comment (jrcastro2) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html:25]: IIRC it was a "select_or_reject" object.

Comment (fenekku) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html:25]: ha, super strange thanks!

**Pull request merged**